pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -469,225 +469,237 @@ func (a *ApplicationBackupController) backupVolumes(backup *stork_api.Applicatio
 	// Start backup of the volumes if we don't have any status stored
 	pvcMappings := make(map[string][]v1.PersistentVolumeClaim)
 	backup.Status.Stage = stork_api.ApplicationBackupStageVolumes
-
-	objectMap := stork_api.CreateObjectsMap(backup.Spec.IncludeResources)
-	info := stork_api.ObjectInfo{
-		GroupVersionKind: metav1.GroupVersionKind{
-			Group:   "core",
-			Version: "v1",
-			Kind:    "PersistentVolumeClaim",
-		},
-	}
-
-	var pvcCount int
-	for _, namespace := range backup.Spec.Namespaces {
-		pvcList, err := core.Instance().GetPersistentVolumeClaims(namespace, backup.Spec.Selectors)
-		if err != nil {
-			return fmt.Errorf("error getting list of volumes to migrate: %v", err)
+	namespacedName := types.NamespacedName{}
+	if IsVolsToBeBackedUp(backup) {
```

Comment:
This he has done because, he want to re-use the code that will move the status to ApplicationBackupStageApplications and call the backupResources(). We can not return nil from here.

Reply:
Yup returning nil will not backupresource for which we want to skip backingup vols.